src/10_Introduction/Internationalization.html

```diff
@@ -0,0 +1,108 @@
+<!---{
+  "hideCode": true,
+  "disablePlayground": true,
+  "hidePreview": true
+}
+--->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/introduction/internationalization/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+<!--
+
+## Introduction
+
+How to support hreflang and internationalization with AMP.
+
+## About Internationalization
+
+Internationalization and localization of content is an important feature that
+websites often embrace to provide maximum reach for their content or services.
+
+For static websites, a common site structure technique for addressing this is
+known as 'hreflang'. Hreflang is a part of the meta data of a website and is used
+to annotate a page such that crawlers can discover alternative translations of
+a particular article. Search engines such as Google recognise websites that use hreflang.
+
+The hreflang notation is used with the link tag to refer to another document:
+
+`<link rel="alternate" hreflang="fr" href="/path/to/french.html" />`
+
+You can [read more about hreflang here.](https://support.google.com/webmasters/answer/189077?hl=en)
+
+AMP documents can also use the hreflang notation to point crawlers towards
+other versions of themselves in other languages.
+
+However, it's important to correctly setup the hreflang relationships between
+AMP documents correctly. Especially when one considers the existing meta data
+relationships between Canonical documents and AMP documents. Below are two examples
+of how websites might structure their hreflang relationships for Canonical and AMP
+documents across an article.
+
+## Example 1: Canonical + AMP
+
+The most common site structure pattern is to support translations of an article
+with a canonical document and a corresponding AMP document.
```

Comment:
Suggestion: rephrase to to

"A common site structure pattern is pairing non-AMP canonical documents with AMP documents."